extensions/import/coverageConfig.json

```diff
@@ -0,0 +1,16 @@
+{
+	"enabled": false,
```

Comment:
Why do you have two files? Get rid of this one

Reply:
I just copied the files over from dacpac extension. I'll remove it here